docs/user-guides/federation.md

```diff
@@ -0,0 +1,165 @@
+# Federate Monitoring Stack and Openshift In-Cluster Prometheus
+
+## Architecture / Topology
+
+![Architecture](federation/assets/cmo-obo-federation.png)
+
+This example deploy a MonitoringStack in `federate-cmo` namespace and reads
```

Comment:
Not a native speacker, but what about?
 > This example deploys a Monitoring stack in the `federate-cmo` namespace...

and maybe Prometheus with capital letter?

Reply:
+1, I will change that